services/gzip.service.js

```diff
@@ -0,0 +1,49 @@
+const request = require('request');
+const zlib = require('zlib');
+const logNestedSiteMap = require('debug')('gzipservice:urls')
+
+const gzipArrs = async (fromArr) => {
+    let toArr = []
+    for (const siteMapUrl of fromArr) {
```

Comment:
this for looks it could have been a map function which is a bit more natural in my opinion. but it's a matter of taste